security/vTPM-tests.py

```diff
@@ -0,0 +1,84 @@
+#!/usr/bin/env python
+# This program is free software; you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation; either version 2 of the License, or
+# (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.
+#
+# See LICENSE for more details.
+#
+# Copyright: 2021 IBM.
+# Author: Nageswara R Sastry <rnsastry@linux.ibm.com>
+
+import os
+from avocado import Test
+from avocado.utils import dmesg, genio, linux_modules
+
+
+class PerfBasic(Test):
```

Comment:
can you please rename this as map like `VtmTest(Test)` , even change test file  name also as `vTmtest.py`

Reply:
Sure, will change. Thanks!!